README.md

```diff
@@ -29,7 +37,8 @@ d
 n
 => default = 1
 => default
-=> +1G
+# Swap partition can be between 100M and 550M
+=> +550M
```

Comment:
The partition mentioned is not swap but EFI, this partition is meant for your boot loader.

Rename to EFI partition and I can accept your changes.

Reply:
FIxed the wrong partition naming